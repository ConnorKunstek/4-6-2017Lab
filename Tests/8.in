SDSSSSDDDDDSSSSSSSSSSSDSSSSSSS
DDDDDDDDDSSSSSSSSSSSSSSDSSSSSS
DSSSSSSSSSSSFDDDDDSSSSSSDSSSSS
DDDDDDDDDDDSSSSSSDSSSSSDSSSSSS
SSSSSSSSSSSSSSFDDDSSSSDSSSSSSS
SSSSSSSSSSSSSSSSSDDDDDSSSSSSSS
SSSSSSSSSFDDDDDDFSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSDDDDDDSSSSSSSDDDDDSSSSSSSSS
SSSDDDDDDSSSSSSSDDDDDSSSSSSSSS
SSSDDDDDDSSSSSSSDDDDDSSSSSSSSS
SSSDDDDDDSSSSSSSSSSSSSSSSSSSSS
DDDDSSSSSSSSSSSSSSSSSSDDDDDDDD
SSSSSSSSSDSSSSSSSSSSSSDSSSSSSS
SSSSSSSSSDSSSSSSSSSSSSDSSSSSSS
SSSSSSSSSDDSSSSSSSSSSSDSSSSSSS
14 6
