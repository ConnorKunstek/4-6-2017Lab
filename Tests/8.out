Yes
Time: 0.00422 seconds

